### PR TITLE
CMA create new assembly for previous RN versions

### PR DESCRIPTION
**Pull request by mburke5678**
This PR creates a new assembly for previous versions of the Custom Metrics Autoscaler Operator release notes. 

Modeled after the Observability docs: [Release notes for past releases of Red Hat build of OpenTelemetry](https://docs.openshift.com/container-platform/4.15/observability/otel/otel_rn/otel-rn-past-releases.html)

**PEER REVIWER** All text has been previously reviewed. The new text in [Custom Metrics Autoscaler Operator release notes](https://74227--ocpdocs-pr.netlify.app/openshift-rosa/latest/nodes/cma/nodes-cma-rn/nodes-cma-autoscaling-custom-rn) was reviewed in [PR73952](https://github.com/openshift/openshift-docs/pull/73952).  The text in [Release notes for past releases](https://74227--ocpdocs-pr.netlify.app/openshift-rosa/latest/nodes/cma/nodes-cma-rn/nodes-cma-autoscaling-custom-rn-past) was reviewed with each release. 
This PR needs **_only a sanity check_** related to breaking the release notes into two assemblies. 

Preview:
https://74227--ocpdocs-pr.netlify.app/openshift-enterprise/latest/nodes/cma/nodes-cma-rn/nodes-cma-autoscaling-custom-rn

### CODE REVIEW

File: nodes/cma/nodes-cma-rn/nodes-cma-autoscaling-custom-rn-past.adoc [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+
+[id="autoscaling-custom-2-8-2-upgrade-operator"]
+==== Operator upgrade support
+
+You can now upgrade from a prior version of the Custom Metrics Autoscaler Operator. See "Changing the update channel for an Operator" in the "Additional resources" for information on upgrading an Operator.
+
+[id="autoscaling-custom-2-8-2-must-gather"]
+==== must-gather support
+
+You can now collect data about the Custom Metrics Autoscaler Operator and its components by using the {product-title} `must-gather` tool. Currently, the process for using the `must-gather` tool with the Custom Metrics Autoscaler is different than for other operators. See "Gathering debugging data in the "Additional resources" for more information.
```
[the remark this answers is not in the document]
Comment (jkyros) [reply nodes/cma/nodes-cma-rn/nodes-cma-autoscaling-custom-rn-past.adoc:141]: this looks like the "downside" of this approach -- our previous release notes will now be subject to current review guidelines (and "bot shaming", apparently) :smile:

File: nodes/cma/nodes-cma-rn/nodes-cma-autoscaling-custom-rn.adoc [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+[IMPORTANT]
+====
+Before installing this version of the Custom Metrics Autoscaler Operator, remove any previously installed Technology Preview versions or the community-supported version of KEDA.
+====
+
+[id="nodes-pods-autoscaling-custom-rn-2121-376-bugs_{context}"]
+=== Bug fixes
+
+* Previously, if invalid values, such as nonexistent namespaces, were specified in scaled object metadata, the underlying scaler clients would not free/close their client descriptors, resulting in a slow memory leak. This fix properly closes the underlying client descriptors when there are errors, preventing memory from leaking. (link:https://issues.redhat.com/browse/OCPBUGS-30145[*OCPBUGS-30145*])
```
Comment (adellape) [inline nodes/cma/nodes-cma-rn/nodes-cma-autoscaling-custom-rn.adoc:59]:
```suggestion
* Previously, if invalid values, such as nonexistent namespaces, were specified in scaled object metadata, the underlying scaler clients would not free or close their client descriptors, resulting in a slow memory leak. This fix properly closes the underlying client descriptors when there are errors, preventing memory from leaking. (link:https://issues.redhat.com/browse/OCPBUGS-30145[*OCPBUGS-30145*])
```

**Review by adellape — COMMENTED** [review]
LGTM re: breaking up the content.

Added a few comments for previously-reviewed content you can take-or-leave in-line and in the related https://github.com/openshift/openshift-docs/pull/73952#pullrequestreview-1997627120.

**Pull request merged**